test/system/080-pause.bats

```diff
@@ -86,4 +86,48 @@ load helpers
 
     run_podman rm -t 0 -f $cname $cname_notrunning
 }
+
+# bats test_tags=ci:parallel
+@test "podman pause/unpause with HealthCheck interval" {
+    if is_rootless && ! is_cgroupsv2; then
+        skip "'podman pause' (rootless) only works with cgroups v2"
+    fi
+
+    local rootless_flag=""
+    if is_rootless; then
+        rootless_flag="--user"
+    fi
```

Comment:
you should import the systemd helpers instead.
```
load helpers.systemd
```
at the top of the file

then `systemctl` is already defined as function which takes care of the user handling